iocore/net/QUICNet.cc

```diff
@@ -142,6 +157,16 @@ QUICPollCont::pollEvent(int, Event *)
   SList(QUICPollEvent, alink) aq(inQueue.popall());
   Queue<QUICPollEvent> result;
   while ((e = aq.pop())) {
+    QUICNetVConnection *qvc = static_cast<QUICNetVConnection *>(e->con);
+    UDPPacketInternal *p    = e->packet;
+    if (qvc != nullptr && qvc->in_closed_queue) {
+      // A server that discards a packet that cannot be associated with a connection MAY also generate a stateless reset (Section
+      // 7.9.4).
+      // Note: Bad, we can't send stateless reset since QUICPollCont doesn't have packet handler.
```

Comment:
I don't think we need to do this here. It's just a "MAY". We already did our best in `QUICPacketHandlerIn`.